internal/backends/sqlite/database.go

```diff
@@ -188,12 +188,9 @@ func relationStats(ctx context.Context, db *fsql.DB, list []*metadata.Collection
 		args[i] = c.TableName
 	}
 
-	// Use number of cells to approximate total row count,
-	// see https://www.sqlite.org/dbstat.html and https://www.sqlite.org/fileformat.html.
 	q = fmt.Sprintf(`
 		SELECT
-		    SUM(pgsize) AS SizeTables,
-		    SUM(ncell)  AS CountCells
+		    SUM(pgsize) AS SizeTables
```

Comment:
The size of table includes all pagetypes, `internal` `overflow` and `leaf` because after all they make up the table.